app/services/file_manager/file_manifests.py

```diff
@@ -121,16 +164,22 @@ def convert_export(attach_post: dict):
 
     def validate_manifest(self, manifest, raise_error=True):
         manifest_validation_event = {'manifest_json': manifest}
-        validation = self.validate_template(manifest_validation_event)
-        if not validation[0]:
-            validation_result = validation[1].get('error_msg').split(' ')
-            error_attr = '_'.join(validation_result[:-1]).upper()
-            validation_error = getattr(ECustomizedError, error_attr)
-            SrvErrorHandler.customized_handle(validation_error, raise_error, validation_result[-1])
-        else:
-            validation = [True]
-            validation_error = ''
-        return validation, validation_error
+        result, validation_error = '', None
+        try:
+            res = self._post('validate/manifest', json=manifest_validation_event)
+            message_handler.SrvOutPutHandler.file_manifest_validation(res.status_code == 200)
+            result = res.json()
+        except Exception as e:
+            response = e.response
+            if response.status_code == 403:
```

Comment:
did we miss the validation error for this one?

Reply:
oh not really this function is not used anymore. But I intend to keep it, in case we need it later on. And by the time, I will reevaluate the cli logic and bff-cli api logic. So for now, I just add some general error handling.